pkg/registry/populator.go

```diff
@@ -119,6 +156,44 @@ func (i *DirectoryPopulator) loadManifests(imagesToAdd []*ImageInput, mode Mode)
 		// same package linearly.
 		var err error
 		var validImagesToAdd []*ImageInput
+		overwriteBundle := ImageInput{}
+
+		// find the image to overwrite if it exists
+		if i.overwrite {
+			for j, image := range imagesToAdd {
```

Comment:
are we no longer constraining our input to a single bundle when `--overwrite-latest` has been specified?